js-core/homeworks/homework-6/src/main.js

```diff
@@ -337,13 +353,14 @@ let listOfCompanys = [
 	return result;
   }
 
+
   function reduceResult(arr){
-	return arr.reduce(function(newValue,value){
-		if(value==`a` ){
-			newValue++;
-		}
-		return newValue;	
-	},0)
+		return arr.reduce(function(newValue,value){
```

Comment:
please make 2 spaces at indention.

Or install [prettier](https://github.com/prettier/prettier) and format your code with it